src/App.js

```diff
@@ -178,19 +169,21 @@ const DonateFormPage1 = ({ baseAmount, handleInputChange }) => {
 
   const amountAsStringIfAny = donation => donation ? donation : '--';
   return (
-    <div>
-      <form>
-        <fieldset>
-          <div>
+    <div className="DonateFormPage1__wrapper">
+        <div className="DonateFormPage1__donationWrapper">
             <label>How much do you want to donate monthly?</label>
-            <span>$</span>
-            <input
-              name="baseAmount"
-              type="text"
-              value={baseAmount}
-              onChange={handleInputChange}
-            />
-            <Link to="/donate/step/2">Next</Link>
+              <div className="DonateFormPage1__inputWrapper"> $
+                <input
+                  name="baseAmount"
+                  type="text"
+                  value={baseAmount}
+                  onChange={handleInputChange}
+                  className="DonateFormPage1__input"
+                />
+              </div>
+            <div className="DonateFormPage1__CTA">
+              <Link to="/donate/step/2">Next</Link>
+            </div>
```

Comment:
If you're using atom I would install https://github.com/prettier/prettier. It reformats automatically on save so that noone has to think about formatting ever again.